android/res/layout/fragment_editor_viral.xml

```diff
@@ -0,0 +1,63 @@
+<?xml version="1.0" encoding="utf-8"?>
+<ScrollView
+  xmlns:android="http://schemas.android.com/apk/res/android"
+  xmlns:tools="http://schemas.android.com/tools"
+  android:layout_width="match_parent"
+  android:layout_height="match_parent">
+
+  <LinearLayout
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:gravity="center_horizontal"
+    android:minWidth="@dimen/width_dialog_base"
+    android:orientation="vertical"
+    android:padding="@dimen/margin_base">
+
+    <TextView
+      android:layout_width="wrap_content"
+      android:layout_height="wrap_content"
+      android:layout_marginBottom="@dimen/margin_half_plus"
+      android:fontFamily="@string/robotoMedium"
+      android:text="@string/thank_you"
+      android:textAppearance="@style/MwmTextAppearance.Body1"
+      tools:ignore="UnusedAttribute"/>
+
+    <ImageView
+      android:layout_width="wrap_content"
+      android:layout_height="wrap_content"
+      android:layout_marginBottom="@dimen/margin_half_plus"
+      android:src="@drawable/img_editor_medal"/>
```

Comment:
+
> On Mar 18, 2016, at 5:31 PM, Alexander Marchuk <notifications@github.com> wrote:
> 
> In android/res/layout/fragment_editor_viral.xml <https://github.com/mapsme/omim/pull/2338#discussion_r56664168>:
> 
> > +    android:padding="@dimen/margin_base">
> > +
> > +    <TextView
> > +      android:layout_width="wrap_content"
> > +      android:layout_height="wrap_content"
> > +      android:layout_marginBottom="@dimen/margin_half_plus"
> > +      android:fontFamily="@string/robotoMedium"
> > +      android:text="@string/thank_you"
> > +      android:textAppearance="@style/MwmTextAppearance.Body1"
> > +      tools:ignore="UnusedAttribute"/>
> > +
> > +    <ImageView
> > +      android:layout_width="wrap_content"
> > +      android:layout_height="wrap_content"
> > +      android:layout_marginBottom="@dimen/margin_half_plus"
> > +      android:src="@drawable/img_editor_medal"/>
> Нужно обернуть всё в HeightLimitedFrameLayout, а этот ImageView пометить как сворачиваемый.
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub <https://github.com/mapsme/omim/pull/2338/files/49a98513afd524db41d893d7cec128cfb271218e..b0dcfabe82c6bad90154275d1b20f2651bbdf1b1#r56664168>